core/src/com/mygdx/crazybomber/Player.java

```diff
@@ -1,46 +1,34 @@
-package com.mygdx.crazybomber.model.player;
-
+//TODO: FIX MOVE LATER
+package com.mygdx.crazybomber;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Batch;
+import com.badlogic.gdx.graphics.g2d.Sprite;
 import com.mygdx.crazybomber.model.bomb.Bomb;
-
 import java.util.Stack;
 import java.util.concurrent.Executors;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
 
-public class Player {
+public class Player extends Sprite {
```

Comment:
So for this class, ming made some changes in his branch. Once his pull request goes through, you might need to re-do this class. Thought your pull req would go in first before.